### PR TITLE
Add accessibility page

### PR DESCRIPTION
**Pull request by ardeparedes**

### CODE REVIEW

File: assets/css/pages.css [hunk 1]
```diff
@@ -42590,59 +42590,13 @@ video::cue {
...
-  margin-right: -6px !important;
-}
 ._1lkwexi4 {
   display: grid !important;
   grid-auto-columns: 1fr !important;
   grid-auto-rows: 1fr !important;
   gap: 8px !important;
   grid-template-columns: 1fr 1fr !important;
 }
-@media (min-width: 375px) {
```
Comment (gbertl) [inline assets/css/pages.css:42611]: removed unused ba to? purgecss?

Comment (gbertl) [reply assets/css/pages.css:42611]: pagpurgecss gawin nalang outside tapos append dito yung cleaned na para iwas conflicts

File: landing/accessibility.html [hunk 2]
```diff
@@ -0,0 +1,1134 @@
+<html data-is-hyperloop="true" lang="en" dir="ltr" class="js-focus-visible">
```
Comment (gbertl) [inline landing/accessibility.html:1]: new pages sa root folder muna natin ilagay

File: assets/css/pages.css (hunk 1 above)
Comment (ardeparedes) [reply assets/css/pages.css:42611]: hindi manual ko yan tinanggal kasi duplicate class

**Pull request merged**